libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -889,6 +916,22 @@ void AP_Baro::update(void)
         }
     }
 
+    const uint32_t now_ms = AP_HAL::millis();
+    if (now_ms - _field_elevation_last_ms >= 1000 && fabs(_field_elevation_active-_field_elevation) > 1.0) {
```

Comment:
Why the 1m resoution? and why the time check?

Reply:
This was put in as a general safety but primarily a template for future growth of the algorithm to use an automated height from GPS/EKF origin or Terrain pull.  I figured for now, this would capture the manual user input use case and get us flying in airspaces where ATC is heavily monitoring our altitudes.